### ISSUE TITLE
About the number of point cloud

### ISSUE THREAD

**Issue by FZH1802** [WanquanF/RMA-Net#11]
Your article states that the number of two point clouds is different，but i found when input data,the target point cloud number and the source point cloud number are same. Can the code handle tasks with inconsistent number of point clouds？
Thanks!

**Comment by whitesay** [comment]
some question.The code 
feature_concat = torch.cat((scores_global, phi_src_tgt, global_phi_src_tgt, global_phi_tgt_src, src_embedding),1)

the dim of global_phi_src_tgt and global_phi_tgt_src seems to depend on the points of source/target, respectively.
